tests/test_transforms/test_math/test_scalers_transform.py

```diff
@@ -73,12 +76,12 @@ def normal_distributed_df() -> pd.DataFrame:
     ),
 )
 @pytest.mark.parametrize("mode", ("macro", "per-segment"))
-def test_dummy_inverse_transform_all_columns(normal_distributed_df, scaler, mode):
+def test_dummy_inverse_transform_all_columns(normal_distributed_ts, scaler, mode):
     """Check that `inverse_transform(transform(df)) == df` for all columns."""
     scaler.mode = TransformMode(mode)
-    feature_df = scaler.fit_transform(df=normal_distributed_df.copy())
-    inversed_df = scaler.inverse_transform(df=feature_df.copy())
-    npt.assert_array_almost_equal(normal_distributed_df.values, inversed_df.values)
+    feature_df = scaler.fit_transform(ts=deepcopy(normal_distributed_ts))
```

Comment:
It is better to create
```
original_df = normal_distributed_ts.to_pandas()
```
Then apply transform in inplace maner
```
scaler.fit_transform(normal_distributed_ts)
inversed_df = scaler.inverse_transform(normal_distributed_ts).to_pandas()
```

Fix this in the next tests, this should emphasize that transforms work inplace